### PR TITLE
Feature/context menu round off

### PR DESCRIPTION
**Pull request by tilt**
https://www.openproject.org/work_packages/7431

### CODE REVIEW

File: app/assets/stylesheets/content/_work_packages_table.css.sass [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  +fade-out()
+
+#work-packages-table
+  tr.issue
+    &.ng-enter, &.ng-move
+      -webkit-transition: 0.5s linear all
+      -moz-transition: 0.5s linear all
+      -o-transition: 0.5s linear all
+      transition: 0.5s linear all
+      opacity: 0
```
Comment (myabc) [inline app/assets/stylesheets/content/_work_packages_table.css.sass:39]: @tilt I've added Bourbon.io, which provides mixins to help with vendor prefixes. See http://bourbon.io/docs/#transitions

File: app/assets/stylesheets/content/_work_packages_table.css.sass [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    &.ng-enter.ng-enter-active, &.ng-move.ng-move-active
+      opacity: 1
+    &.ng-leave
+      -webkit-animation: 0.5s fade-out
+      -moz-animation: 0.5s fade-out
+      -o-animation: 0.5s fade-out
+      animation: 0.5s fade-out
```
Comment (myabc) [inline app/assets/stylesheets/content/_work_packages_table.css.sass]: @tilt ditto: http://bourbon.io/docs/#animations

**Comment by tilt** [comment]
@myabc Made the whole `<li/>` clickable and used bourbon for the animations. Could you look over and merge this?

File: config/locales/js-en.yml [hunk 3]
```diff
@@ -135,6 +135,7 @@ en:
         one: "You can only select one item"
         other: "You can only select {{limit}} items"
         zero: "You cannot select any items"
+    text_work_packages_destroy_confirmation: "Are you sure you want to delete the selected work package(s)?"
```
Comment (myabc) [inline config/locales/js-en.yml:138]: @tilt I'm nitpicking here (also not a prerequisite for merging this PR now), but could we use `ngPluralize` here to customise the message based on number selected?

**Comment by myabc** [comment]
Awesome! :cool: 

Great that this is working well with flash messages too :-) (the attached error was expected)
seeded_project_-_openproject

**Pull request merged**